ui-packages/packages/management-console/src/components/Atoms/BulkList/BulkList.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+  TextContent,
+  TextVariants,
+  Text,
+  Divider,
+  TextList,
+  TextListItem
+} from '@patternfly/react-core';
+import {
+  ItemDescriptor,
+  OUIAProps,
+  componentOuiaProps
+} from '@kogito-apps/common';
+
+export enum OperationType {
+  ABORT = 'ABORT',
+  SKIP = 'SKIP',
+  RETRY = 'RETRY',
+  CANCEL = 'CANCEL'
+}
+
+export enum BulkListType {
+  PROCESS_INSTANCE = 'process_instnace',
+  JOB = 'job'
+}
+export interface IOperationResult {
+  successItems: BulkListItem[];
+  failedItems: BulkListItem[];
+  ignoredItems: BulkListItem[];
+}
+export interface IOperationMessages {
+  successMessage: string;
+  warningMessage?: string;
+  ignoredMessage: string;
+  noItemsMessage: string;
+}
+
+export interface IOperationFunctions {
+  perform: () => void;
+}
+
+export interface IOperationResults {
+  [key: string]: IOperationResult;
+}
+
+export interface IOperation {
+  type: string;
+  results: IOperationResult;
+  messages: IOperationMessages;
+  functions: IOperationFunctions;
+}
+
+export interface IOperations {
+  [key: string]: IOperation;
+}
+export interface IOwnProps {
+  operationResult: IOperation;
+}
+export interface BulkListItem {
+  id: string;
+  name: string;
+  description: string;
+  errorMessage?: string;
+}
+
+const BulkList: React.FC<IOwnProps & OUIAProps> = ({
+  operationResult,
+  ouiaId,
+  ouiaSafe
+}) => {
+  const iterateItems = itemList => {
+    return (
+      <TextList>
+        {itemList.map((item: BulkListItem) => {
+          return (
+            <TextListItem key={item.id}>
+              <strong>
+                <ItemDescriptor itemDescription={item} />
+              </strong>{' '}
+              {item.errorMessage && <span> - {item.errorMessage}</span>}
+            </TextListItem>
+          );
+        })}
+      </TextList>
+    );
+  };
+  return (
+    <div {...componentOuiaProps(ouiaId, 'bulk-list-instances', ouiaSafe)}>
```

Comment:
I would prefer the ouia type to be in sync with component name.
```suggestion
    <div {...componentOuiaProps(ouiaId, 'bulk-list', ouiaSafe)}>
```

Reply:
sure, done :+1: